CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ PROJECT(MojoShader)
 OPTION(BUILD_SHARED_LIBS "Build MojoShader as a shared library" OFF)
 OPTION(PROFILE_D3D "Build MojoShader with support for the D3D profile" ON)
 OPTION(PROFILE_BYTECODE "Build MojoShader with support for the BYTECODE profile" ON)
+OPTION(PROFILE_HLSL, "Build MojoShader with support for the HLSL profile" ON)
```

Comment:
Happened to run into the comma after `PROFILE_HLSL` by accident when compiling testparse...